templates/e-commerce/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,18 @@
+{{#hero}}
+<section class="commerce-hero-content center">
+  {{#images}}
+    {{#image}}
+      <amp-img src="{{url}}" width="{{width}}" height="{{height}}" layout="responsive" media="{{media}}" alt="{{alt}}"></amp-img>
+    {{/image}}
+  {{/images}}
+  <div class="commerce-hero-content-content commerce-hero-content-theme-{{text-theme}} flex justify-center flex-column">
```

Comment:
Renamed.

The theme is dependant on the colour of the background image which can be in any order, so  it's needed as a class.